docs/docsite/rst/dev_guide/style_guide/index.rst

```diff
@@ -183,7 +183,7 @@ Long pages, or pages with multiple levels of headings, can also include a local
 
 .. note::
 
-	Avoid raw URLs. RST and sphinx allow ::code:`https://my.example.com`, but this is unhelpful for those using screen readers. ``:ref:`` links automatically pick up the heading from the anchor, but for external links, always use the ::code:`link title <link-url>`_` format.
+	Avoid raw URLs. RST and sphinx allow ::code:`https://my.example.com`, but this is unhelpful for those using screen readers. ``:ref:`` links automatically pick up the heading from the anchor, but for external links, always use the ::code:`\`link title <link-url>\`_` format.
```

Comment:
```suggestion
	Avoid raw URLs. RST and sphinx allow :code:`https://my.example.com`, but this is unhelpful for those using screen readers. ``:ref:`` links automatically pick up the heading from the anchor, but for external links, always use the :code:`\`link title <link-url>\`_` format.
```

Reply:
looking at the rendered preview, there were extra colons (in the original) so would be great to fix that in this pr. Thanks!